src/base/stats/units.hh

```diff
@@ -174,6 +174,23 @@ class Second : public Base
     std::string getUnitString() const override { return Second::toString(); }
 };
 
+class NanoSecond : public Base
```

Comment:
@powerjg If I want to express latency stats in seconds, due to precision limitations they will all be zero. I added this new unit to express the latencies in nanoseconds. I am not really sure if this is what the right way to do this is. Let me know if I have to change this to Tick.